src/iuse.cpp

```diff
@@ -2305,13 +2304,7 @@ int iuse::hammer( player *p, item *it, bool, const tripoint & )
         return 0;
     }
 
-    ter_id type = g->m.ter( pnt );
-    if( pry_nails( *p, type, pnt ) ) {
-        return it->type->charges_to_use();
-    } else {
-        p->add_msg_if_player( m_info, _( "There's nothing to pry there." ) );
-    }
-    return 0;
+    return this->crowbar( p, it, false, pnt );
```

Comment:
Unless I'm mistaken, this will cause doubling of `"Pry where?"` request? First time when when activating the hammer, and the second time when calling `crowbar` function.

Reply:
`hammer(...)` checks if the player cancelled the pry request and exits, then checks if the player targeted themselves, prints an amusing message then exits.

If we make it past both those conditionals, it calls `crowbar(...)` passing the selected position (which is not the players position) into `crowbar(...)` which then checks gain if the target was the player object's position.
If the target was the player, you get another chance to select a pry location (so using a crowbar directly still works as intended) - and then another amusing message if they persist in selecting themselves.

But calling from hammer should skip all that as the player has to target something other than themselves to get to the crowbar invocation.